src/integrationTest/resources/application.properties

```diff
@@ -9,6 +9,13 @@ envelope-access.mappings[0].jurisdiction=BULKSCAN
 envelope-access.mappings[0].readService=test_service
 envelope-access.mappings[0].updateService=update_service
 
+containers.mappings[0].container=bulkscan
+containers.mappings[0].jurisdiction=BULKSCAN
+containers.mappings[0].poBox=SSCSPO
```

Comment:
maybe BULKSCANPO would be clearer

Reply:
@rkondratowicz Updated properties. I had to change the metadata files to fix integration tests